src/distributors/adoptopenjdk-installer.ts

```diff
@@ -0,0 +1,77 @@
+import * as core from '@actions/core';
+import * as tc from '@actions/tool-cache';
+
+import fs from 'fs';
+import path from 'path';
+import semver from 'semver';
+
+import { IS_WINDOWS, PLATFORM, IS_MACOS, macOSJavaContentDir } from "../util";
+import { JavaInstallerResults, IJavaRelease, JavaBase, JavaInstallerOptions } from "./base-installer";
+import { IRelease, IReleaseVersion } from './adoptopenjdk-models'
+
+export class AdoptOpenJDKDistributor extends JavaBase {
+    
+    constructor(initOptions: JavaInstallerOptions) {
+        super("AdoptOpenJDK", initOptions);
+    }
+
+    private async resolveMajorVersion(range: semver.Range) {
```

Comment:
could you please sort class methods the following way:
- public
- protected
- private

Please do this in adopt and azul classes

Reply:
Done.